tests/Localization/LtLtTest.php

```diff
@@ -17,210 +17,415 @@ class LtLtTest extends LocalizationTestCase
+        // 'Yesterday at 10:00 PM'
         'Yesterday at 22:00',
+
         // Carbon::parse('2018-01-04 12:00:00')->subHours(2)->calendar(Carbon::parse('2018-01-04 12:00:00'))
+        // 'Today at 10:00 AM'
         'Today at 10:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->addHours(2)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Today at 2:00 AM'
         'Today at 02:00',
+
         // Carbon::parse('2018-01-04 23:00:00')->addHours(2)->calendar(Carbon::parse('2018-01-04 23:00:00'))
+        // 'Tomorrow at 1:00 AM'
         'Tomorrow at 01:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->addDays(2)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Tuesday at 12:00 AM'
         'antradienis at 00:00',
+
         // Carbon::parse('2018-01-08 00:00:00')->subDay()->calendar(Carbon::parse('2018-01-08 00:00:00'))
+        // 'Yesterday at 12:00 AM'
         'Yesterday at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(1)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Yesterday at 12:00 AM'
         'Yesterday at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(2)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Tuesday at 12:00 AM'
         'Last antradienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(3)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Monday at 12:00 AM'
         'Last pirmadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(4)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Sunday at 12:00 AM'
         'Last sekmadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(5)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Saturday at 12:00 AM'
         'Last šeštadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Friday at 12:00 AM'
         'Last penktadienis at 00:00',
+
         // Carbon::parse('2018-01-03 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-03 00:00:00'))
+        // 'Last Thursday at 12:00 AM'
         'Last ketvirtadienis at 00:00',
+
         // Carbon::parse('2018-01-02 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-02 00:00:00'))
+        // 'Last Wednesday at 12:00 AM'
         'Last trečiadienis at 00:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->subDays(2)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Last Friday at 12:00 AM'
         'Last penktadienis at 00:00',
+
         // Carbon::parse('2018-01-01 00:00:00')->isoFormat('Qo Mo Do Wo wo')
+        // '1st 1st 1st 1st 1st'
         '1 1 1 1 1',
+
         // Carbon::parse('2018-01-02 00:00:00')->isoFormat('Do wo')
+        // '2nd 1st'
         '2 1',
+
         // Carbon::parse('2018-01-03 00:00:00')->isoFormat('Do wo')
+        // '3rd 1st'
         '3 1',
```

Comment:
`3-ias 1-as`

Reply:
`3-ias` is the only exception, all other ordinal numbers are `N-as`?

Reply:
i guess so because it's ending is `ias`, others have `as`
also 0 is 0-is because nul**is**

Reply:
Updated.